cmd/token.go

```diff
@@ -101,11 +106,22 @@ The OIDC token includes claims regarding the GCE VM, which is verified by Attest
 		}
 		gceAK.Close()
 
+		cloudLogEnabled, _ := cmd.Flags().GetBool("cloud-log")
+		if cloudLogEnabled {
+			audience = "Access and Audit"
+			cloudLogClient, err = logging.NewClient(ctx, projectID)
+			if err != nil {
+				return fmt.Errorf("failed to create Cloud Logging client: %w", err)
+			}
+			cloudLogger = cloudLogClient.Logger(toolName)
+			fmt.Fprintf(debugOutput(), "cloudLogger created for project: "+projectID+"\n")
+		}
+
```

Comment:
We shouldn't override audiences the user sets. Just check 
```
if cloudLog && audience == "" {
  return errors.New("... ")
}
```

Reply:
Yes, we should not. I set the default audience just now to "https://api.example.com" and forgot to remove the override here. Thanks for pointing it out! And please let me know if you have other thoughts about default audience value.

Reply:
the default should be an empty audience, so that users don't accidentally issue a token with https://api.example.com/ by default. An empty audience should mean we don't fill out the audience field